components/Sponsors.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import { H1, Section } from './style'
-import { Style as StyleLogoGrid } from './LogoGrid'
+import { Style as Style_LogoGrid } from './LogoGrid'
```

Comment:
Please do not use `Style_LogoGrid`.  This is the wrong style of variable.

Reply:
I used StyleLogoGrid first but I think you changed it to Style_LogoGrid, if it's the wrong style of variable, what should I use?

Reply:
I didn't change the style to `Style_LogoGrid` -- you did in a previous PR.  Note that it is set to `Style_LogoGrid` in the current version and I don't know why you are changing it.